atom_calibration/src/atom_calibration/collect/data_collector.py

```diff
@@ -94,11 +96,12 @@ def __init__(self, args, server, menu_handler):
         self.bridge = CvBridge()
         self.dataset_version = "3.0"  # included joint calibration
         self.joint_state_position_dict = {}
-
-        # print(args['calibration_file'])
+        self.rospack = rospkg.RosPack()
```

Comment:
Is this variable used anywhere? From my brief search I could not find any reference to this.

Reply:
I don't think so, I will remove it.